applications/FluidDynamicsApplication/custom_processes/distance_smoothing_process.h

```diff
@@ -70,32 +92,188 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) DistanceSmoothingProcess : public P
     /// Constructor.
     DistanceSmoothingProcess(
         ModelPart& rModelPart,
-        typename TLinearSolver::Pointer);
+        typename TLinearSolver::Pointer p_linear_solver,
```

Comment:
```suggestion
        typename TLinearSolver::Pointer pLinearSolver,
```
[Style guide ](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide)

Reply:
Done.